src/main/java/it/gov/pagopa/pu/debtpositions/service/massive/DebtPositionMassiveService.java

```diff
@@ -0,0 +1,8 @@
+package it.gov.pagopa.pu.debtpositions.service.massive;
+
+import it.gov.pagopa.pu.debtpositions.dto.generated.InstallmentSynchronizeDTO;
+
+public interface DebtPositionMassiveService {
```

Comment:
Don't call it "Massive" actually it's an API just used by the massive flow, but actually it could be invoked by any client (change also the package)
```suggestion
public interface InstallmentSynchronizationService {
```